packages/icontools/package.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+#     spack install icontools
+#
+# You can edit this file again by typing:
+#
+#     spack edit icontools
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack import *
+import subprocess
+
+class Icontools(AutotoolsPackage):
+    """
+    DWD ICON Tools for C2SM members. 
+    Set of tools to prepare the input files 
+    (for example the boundary condition, initial condition file,...) for ICON.
+    """
+
+    homepage= 'https://wiki.c2sm.ethz.ch/MODELS/ICONDwdIconTools'
+    git = 'git@github.com:C2SM-ICON/dwd_icon_tools.git'
+
+    maintainers = ['jonasjucker']
+
+    version('master', branch='master')
+
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool',  type='build')
+    depends_on('m4', type='build')
+
+    depends_on('cray-libsci %cce', type=('build', 'link'))
+    depends_on('netcdf-fortran %cce ~mpi', type=('build', 'link'))
+    depends_on('netcdf-c %cce ~mpi', type=('build', 'link'))
+    depends_on('mpi', type=('build', 'link', 'run'),)
```

Comment:
Might this package also be used on Tsa? Otherwise you have to do the mpi, mpicuda trick (see cosmo), that is because we have to special modules on Tsa, which is bad but we found no alternative...

Reply:
I need to talk with Guy, once he is back from holiday about it.
For C2SM it is not a goal to have it on Tsa, but if MCH wants it there we need   to make it compatible.

I wrote Guy an email about that, so no decision now...

Reply:
@jonasjucker I don't see any reason not to install on tsa.
I hope the mpicuda trick @elsagermann mentions works here.

Reply:
I didn't needed to apply this trick, since there is no GPU for Icontools.
It works find just using the MPI-dependency.

Did I miss something here?

Reply:
Oh yé yes of course sorry indeed you don't need mpicuda then!